### DISCUSSION TITLE
增加自定义作者页面封面

### DISCUSSION THREAD

**Discussion by Lorikein12138** [LIlGG/halo-theme-sakura#511] [General]
### 目前使用的 Sakura 版本？

2.1.0

### 目前使用的 halo 版本？

2.15.0

### 建议/问题

新增加的作业页面的封面只能通过超级管理员在后台设置
下面是普通用户的个人资料修改
![image](https://github.com/LIlGG/halo-theme-sakura/assets/117804471/f2fadd33-3b91-43a9-b7e7-10490e91458b)
下面是超级管理员通过后台用户修改个人资料
![image](https://github.com/LIlGG/halo-theme-sakura/assets/117804471/efb31dc1-ebc4-439b-bc9c-84345acc4c06)

普通用户无法自己修改封面
希望增加这个功能

**Comment by LIlGG** [comment]
目前 Halo 不支持在个人中心的文章中显示元数据，因此这个功能需要 Halo 进行支持。详见 https://github.com/halo-dev/halo/issues/4953
